buildkite/src/Constants/DockerPublish.dhall

```diff
@@ -0,0 +1,46 @@
+let Artifacts = ./Artifacts.dhall
+
+let DockerPublish
+    : Type
+    = < Enabled | Disabled | Essential >
```

Comment:
Is my understanding correct that `Enabled` isn't used now and is primarily for the possible future usage?